docs/further.md

```diff
@@ -160,6 +160,15 @@ set-resources:
         cpus_per_task: 40
 ```
 
+### Additional Command Line Flags
+
+This plugin defines additional command line flags. As always the can be used on the command line or in a profile.
```

Comment:
Mini rephrasing of the second sentence. And I prefer having one sentence per line in Markdown files, for clearer git diffs down the line...
```suggestion
This plugin defines additional command line flags.
As always, these can be set on the command line or in a profile.
```